mmv1/products/securitycenter/api.yaml

```
@@ -66,3 +66,94 @@ objects:
           can't share the same display name. The display name must start and end
           with a letter or digit, may contain letters, digits, spaces, hyphens,
           and underscores, and can be no longer than 32 characters.
+  - !ruby/object:Api::Resource
+    name: 'NotificationConfig'
+    base_url: organizations/{{organization}}/notificationConfigs
+    self_link: '{{name}}'
+    create_url: organizations/{{organization}}/notificationConfigs?configId={{config_id}}
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      A Cloud Security Command Center (Cloud SCC) notification configs. A
+      notification config is a Cloud SCC resource that contains the
+      configuration to send notifications for create/update events of
+      findings, assets and etc.
```

Hi @Erouan50, our testing failed because our test org wasn't enrolled in SCC. Can we add something to the documentation like:
```suggestion
      findings, assets and etc.
      ~> **Note:** In order to use Cloud SCC resources, your organization must be enrolled 
      in [SCC Standard/Premium](https://cloud.google.com/security-command-center/docs/quickstart-security-command-center).
      Without doing so, you may run into errors during resource creation.
```

---

@c2thorn are you talking about activating the SCC API or something else?

---

Not the API specifically, that was pretty standard. I had to follow the steps in https://cloud.google.com/security-command-center/docs/quickstart-security-command-center to enroll our org. Otherwise I was receiving `Error 400: Request contains an invalid argument.`